alembic/versions/82c1df0542a7_fix_bracket_id_for_user_tournament.py

```diff
@@ -0,0 +1,30 @@
+"""fix bracket_id for user_tournament
+
+Revision ID: 82c1df0542a7
+Revises: 9bea906e3151
+Create Date: 2022-01-11 08:40:00.119979
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '82c1df0542a7'
+down_revision = '9bea906e3151'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint('user_tournaments_bracket_id_fkey', 'user_tournaments', type_='foreignkey')
+    op.create_foreign_key(None, 'user_tournaments', 'brackets', ['bracket_id'], ['id'])
```

Comment:
did you test this? Does it work?

Reply:
Wait, what is this fixing? Shouldn't the user_tournaments table be a join table on users and tournaments?